src/backend/query_builder.rs

```diff
@@ -788,6 +790,10 @@ pub trait QueryBuilder: QuotedBuilder {
             Value::DateTimeWithTimeZone(Some(v)) => {
                 write!(s, "\'{}\'", v.format("%Y-%m-%d %H:%M:%S %:z").to_string()).unwrap()
             }
+            #[cfg(feature = "with-chrono")]
+            Value::DateTimeUtc(Some(v)) => {
+                write!(s, "\'{}\'", v.format("%Y-%m-%d %H:%M:%S %z").to_string()).unwrap()
+            }
```

Comment:
Wouldn't it be better to state the timezone in the format that same as `Value::DateTimeWithTimeZone`? i.e. `+00:00`

REF: https://docs.rs/chrono/latest/chrono/format/strftime/index.html

Reply:
Sure. I will make the changes